.kitchen.dokken.yml

```diff
@@ -14,6 +14,8 @@ platforms:
   driver:
     image: dokken/amazonlinux
     pid_one_command: /sbin/init
+    intermediate_instructions:
+      - RUN echo NETWORKING=yes > /etc/sysconfig/network
```

Comment:
This breaks things. I've added and removed this line twice now. From here and Haproxy